src/main/java/ToDo.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Todo class, subclass of Task
+ */
+public class ToDo extends Task {
+
+    /**
+     * Constructor for Todo class
+     * @param description
+     */
+    public ToDo(String description){
+        super(description);
+    }
+
+    /**
+     * String representation of Todo instance
+     * @return
+     */
+    @Override
+    public String toString() {
+        return "[T]" + super.toString() ;
```

Comment:
Small nit. Perhaps we can remove the extra spacing after the toString() call.
```suggestion
        return "[T]" + super.toString();
```